burotel/indico_burotel/client/js/components/BootstrapOptions.jsx

```diff
@@ -15,7 +15,7 @@ import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
 import {Translate} from 'indico/react/i18n';
 
 
-export const EXPERIMENTS = ['ATLAS', 'CMS', 'ALICE'];
+export const EXPERIMENTS = ['ATLAS', 'CMS', 'ALICE', "LHCb"];
```

Comment:
You like mixing quotation marks? :P

Reply:
CRAPZOR. How did I not see that?

![image](https://user-images.githubusercontent.com/2699/55554694-b4a83c00-56e3-11e9-96ad-5e8215275645.png)